fastlane/Fastfile

```diff
@@ -273,7 +273,6 @@ REPOSITORY_NAME="WordPress-Android"
     android_finalize_prechecks(options)
     configure_apply(force: is_ci)
     hotfix = android_current_branch_is_hotfix
-    android_update_metadata(options) unless hotfix
```

Comment:
Any reason why we don't download the strings in `finalize_release` anymore? Seems we could easily missed some of the latest translations since last beta if we remove that line…

Reply:
In fact, checking out your branch and running `bundle exec fastlane download_translations` shows thet there are some strings being updated

Reply:
Ahhhh! thanks for catching this this was not suppose to be removed but replaced with the new download_translations lane

Reply:
@JavonDavis – does this mean we missed new translations in the most recent build?

Reply:
I'll let Javon reply in details and confirm, but yes we missed some indeed in the build submitted on Friday. Which I believe have since been fixed via 8be28f9 then 6bbc145 to re-download the latest translations 2 hours ago, which means that this PR should now be up-to-date with those latest translations, but that we'll still require to submit a new build for 17.3 and go thru the validation process again before being able to start rollout iinm.

Reply:
Yep after looking into this more with Olivier we did miss some of the new strings and addressed that in https://github.com/wordpress-mobile/WordPress-Android/pull/14664/commits/6bbc145cd61c50b94e56aab1b00540bd9738c79e and a new build submitted to Google